components/db/testimony/usePublishedTestimonyListing.ts

```diff
@@ -91,16 +104,66 @@ async function listTestimony(
   limitCount: number,
   startAfterKey: unknown | null
 ): Promise<Testimony[]> {
-  const testimonyRef = collectionGroup(firestore, "publishedTestimony")
-  const result = await getDocs(
-    nullableQuery(
-      testimonyRef,
-      ...getWhere(refinement),
-      where("court", "==", currentGeneralCourt),
-      orderBy("publishedAt", "desc"),
-      limit(limitCount),
-      startAfterKey !== null && startAfter(startAfterKey)
-    )
-  )
-  return result.docs.map(d => d.data() as Testimony)
+  const client = createClient()
+  const publishedBool = refinement.published === "true"
+
+  let query = { }
+
+  if (!publishedBool && refinement.billId && refinement.uid) {
+      const result = await getBillTestimony(refinement.uid, refinement.billId)
+
+      return [result.draft] as Testimony[]
+  }
+
+  if (refinement.billId && refinement.uid) {
+    query = {
+      q: `${refinement.uid}, ${refinement.billId}`,
```

Comment:
I think we can use [filter_by](https://typesense.org/docs/0.23.1/api/search.html#query-parameters) on uid and/or billId, and use `q:*` to return everything that matches.

Reply:
Yes, I tried to use it, but couldn't make it work 😅